### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by VEINHORN**

### CODE REVIEW

**Comment by VEINHORN** [comment]
I signed it!

File: android/src/main/java/com/google/samples/apps/iosched/service/SessionCalendarService.java [hunk 1]
```diff
@@ -102,9 +102,7 @@ protected void onHandleIntent(Intent intent) {
                         processAllSessionsCalendar(resolver, getCalendarId(intent)));
                 sendBroadcast(new Intent(
                         SessionCalendarService.ACTION_UPDATE_ALL_SESSIONS_CALENDAR_COMPLETED));
-            } catch (RemoteException e) {
-                LOGE(TAG, "Error adding all sessions to Google Calendar", e);
-            } catch (OperationApplicationException e) {
+            } catch (Exception e) {
```
Comment (mingfai) [inline android/src/main/java/com/google/samples/apps/iosched/service/SessionCalendarService.java]: imho, it is better to change to 
```
    } catch (RemoteException | OperationApplicationException e){
        //...
    } 
```

Comment (eneim) [inline android/src/main/java/com/google/samples/apps/iosched/service/SessionCalendarService.java]: IMO Your approach has a better readability, but it requires higher version of Java (which is not a big problem now, actually). For those who still need to work with Java 6, keeping the old lines is better. And generally, catching a Abstract Exception is not a good practice.

**Comment by piotrek1543** [comment]
@VEINHORN, your Travis config file seems to be outdated. Please update that file. 
Yo can use one of my pull request commits to change it

Cheers!

**Pull request closed without merging**